tests/basic/pool_function_test.go

```diff
@@ -186,3 +189,152 @@ var _ = Describe("{PoolExpandWithPXRestart}", func() {
 		})
 	})
 })
+
+var _ = Describe("{PoolExpandResizeInvalidPoolID}", func() {
```

Comment:
This test can live in Smoky suite https://github.com/portworx/torpedo/blob/e666896c9783adcdcddde0ba8c9b13c3b213fa93/tests/basic/pool_function_test.go#L23

Reply:
lets keep it all tests separate for now,
later we can further divide when we are all complete